app/src/main/java/org/projectbuendia/client/ui/OdkActivityLauncher.java

```diff
@@ -111,40 +142,30 @@ public static void fetchAndShowXform(
                             FetchXformFailedEvent.Reason.NO_FORMS_FOUND));
                         return;
                     }
-                    // Cache all the forms into the ODK form cache
-                    new OdkXformSyncTask(new OdkXformSyncTask.FormWrittenListener() {
-                        @Override public void formWritten(File path, String uuid) {
-                            LOG.i("wrote form " + path);
-                            showOdkCollect(
-                                callingActivity,
-                                requestCode,
-                                OdkDatabase.getFormIdForPath(path),
-                                patient,
-                                fields);
-                        }
-                    }).execute(findUuid(response, uuidToShow));
+                    // Cache the form into the ODK form cache
```

Comment:
Thx @capnfabs. I deleted this comment. The `showForm`'s javacdoc was out of date too. I corrected it and removed an unused param too.